aws/resource_aws_msk_cluster.go

```diff
@@ -434,12 +440,23 @@ func resourceAwsMskClusterRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("failed requesting bootstrap broker info for %q : %s", d.Id(), err)
 	}
 
+	nodesOut, err := conn.ListNodes(&kafka.ListNodesInput{
+		ClusterArn: aws.String(d.Id()),
+	})
+	if err != nil {
+		return fmt.Errorf("failed requesting list nodes info for %q : %s", d.Id(), err)
```

Comment:
small nit, use `%w` for error

Reply:
Thanks, updated in https://github.com/terraform-providers/terraform-provider-aws/commit/f48c1890c4d5a9c972c0a6166970a4ffc1f73272